javascript/src/main/pages/TutorAssignment/TutorAssignments.js

```diff
@@ -5,7 +5,7 @@ import { fetchWithToken } from "main/utils/fetch";
 import { useAuth0 } from "@auth0/auth0-react";
```

Comment:
think there is a merge conflict here, not sure how to resolve, maybe professor or a TA can help

Reply:
I believe I fixed the merge conflict. It passed all tests and worked when I ran locally

Reply:
I am actually confused on how to resolve the merge conflicts. It seems that it should just replace the line that is being replaced, but github is saying it is a merge conflict still

Reply:
Nevermind I was just confused about how github works, i was able to resolve the conflict